ToDo.Core.Test/ToDoViewModelTest.cs

```diff
@@ -97,5 +97,27 @@ public async Task AddNewTodoTest()
             _navigationService.Verify(
                 s => s.Navigate<AddViewModel>(null, default), Times.Once);
         }
+
+        [TestMethod]
+        public async Task EditTodoTest()
```

Comment:
Naming tests is an important task.
Especially if the test is complex, it is very important that the title already suggests what exactly is done.

The tempalte would be something like `public void {precondition}_{dointsomething}_{expectedoutcome}()`;

So here, instead of `EditTodoTest` a better name would be:
`public void WhenEditingTodo_NavigatesToEditViewModel()`

Please rename the test methods as an exercise!